pdfannots/types.py

```diff
@@ -296,6 +296,7 @@ def __init__(
             quadpoints: typ.Optional[typ.Sequence[float]] = None,
             rect: typ.Optional[BoxCoords] = None,
             contents: typ.Optional[str] = None,
+            color: typ.Optional[str] = None,
```

Comment:
1. It's probably safer to put this constructor parameter last, just in case there was some code out there relying on argument order.
2. Please document the new field (in the block comment above)
4. (Optional, but encouraged) if I was writing this, I'd have preferred to keep the colour formally typed, and convert to an RGB hex string only in the output phase. For example, you could declare a type such as:

```
class RGB(typ.NamedTuple):
    red: float
    green: float
    blue: float
```

and then Annotation.color can be  type RGB, which you can create over in the parser almost directly from the floats as `RGB(red, green, blue)`. Finally, the hex formatting code moves out to the json serialiser.

Reply:
I implement RGB class with one method called RGB.ashex()